src/components/ResultsListItem.vue

```diff
@@ -1,10 +1,15 @@
 <template>
   <section class="item-card">
-    <button @click="savePark" class="save-button">
-      SAVE
-    </button>
     <h1>{{ result.name }}</h1>
     <h1>{{ result.formatted_address }}</h1>
+    <p>This park is {{ result.opening_hours.open_now ? 'open' : 'closed' }}</p>
+    <p>Rating: {{ result.rating }}</p>
+    <button v-if="this.$route.path !== '/my-parks'" @click="savePark" class="save-button">
```

Comment:
I love how you used the route's path to ensure that if a user is viewing their saved parks, a `save` button isn't accidentally rendered.